WooCommerce/Classes/ViewRelated/Products/ProductsListViewModel.swift

```diff
@@ -23,6 +23,7 @@ final class ProductListViewModel: ProductsListViewModelProtocol {
 
     private let barcodeSKUScannerItemFinder: BarcodeSKUScannerItemFinder
     private let featureFlagService: FeatureFlagService
+    let favoriteProductsUseCase: FavoriteProductsUseCase
```

Comment:
Super nit: do we want to keep this private as well?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/12287/commits/9b9e11aed86de22a62ceb5e2403c455c816066fe